exercises/twelve-days/description.md

```diff
@@ -1,6 +1,12 @@
-# Description
+# Instructions
```

Comment:
Maybe have this be unchanged to match the other exercises?

```suggestion
# Description
```

Reply:
Ah, that was unintentional